.github/workflows/backend.yml

```diff
@@ -3,8 +3,11 @@ name: Agent CI
 on:
   push:
     branches: [ main ]
+
   pull_request:
     branches: [ main ]
+    paths-ignore:
+      -  'frontend/**'
```

Comment:
Should we also do this for pushes to `main`?

Reply:
As I wrote, I decided against it. But if you convince me I can add it.

Reply:
> I decided to run in on merges to main unconditionally, so we have data there.

Yeah, I did not get this completely, not sure what data you were refering to.
But in any case, it's a minor point, feel free to ignore and merge.